test/clang-tidy/mesos-redundant-get.cpp

```diff
@@ -0,0 +1,98 @@
+// RUN: %check_clang_tidy %s mesos-redundant-get %t
+
+struct Foo {
+  void bar();
+  int baz;
+  void quz() const;
+};
+
+template <typename T>
+struct Future {
+  T *operator->();
+  T& get();
+
+  const T *operator->() const;
+  const T &get() const;
+};
+
+template <typename T>
+using Option = Future<T>;
```

Comment:
How about a test for the other aliasing direction, i.e.

    typedef Foo Try;

Reply:
This is almost guaranteed to be the same as soon as the AST is generated with the expanded types. I added a test nevertheless.